src/App.test.js

```diff
@@ -55,6 +55,17 @@ test("displays correct result of adding -7 to 7", () => {
   expect(evaluate('7+-7')).toBe(0);
 });
 
+test("displays correct result of adding 0.1 and 0.2", () => {
```

Comment:
Why doesn't this test use fireEvent and render App like the others?

Reply:
Ah yes, that was test code that was being used to check if evaluate was working, and to check if using a config for math.js would help with the decimal bug. I removed it as it is no longer necessary.